src/app/design/frontend/base/default/template/payment/form/omisecc.phtml

```diff
@@ -1,77 +1,119 @@
-                    Omise.createToken("card", card, function (statusCode, response) {
-                        // If has an error (can not create a card's token).
-                        if (response.object == "error") {
-                            // Display an error message - "Omise Response: "+response.message.
-                            formValidation.css({'opacity': 1, 'display': 'block'});
-                            formValidationMsg.html("Omise Response: "+response.message);
-
-                            jQuery("#payment-buttons-container").find("button").prop("disabled", false);
-                        } else if (typeof response.card != 'undefined' && !response.card.security_code_check) {
-                            // Display an error message - "Omise Response: Card authorization failure.".
-                            formValidation.css({'opacity': 1, 'display': 'block'});
-                            formValidationMsg.html("Omise Response: Card authorization failure.");
-
-                            jQuery("#payment-buttons-container").find("button").prop("disabled", false);
-                        } else {
-                            form.find("[data-omise=holder_name]").prop("disabled", true);
-                            form.find("[data-omise=number]").prop("disabled", true);
-                            form.find("[data-omise=expiration_month]").prop("disabled", true);
-                            form.find("[data-omise=expiration_year]").prop("disabled", true);
-                            form.find("[data-omise=security_code]").prop("disabled", true);
-
-                            form.find("[data-omise=omise_token]").val(response.id);
-
-                            return superMethod();
-                        };
-                    });
+    if ((typeof jQuery === 'undefined') && ! window.jQuery) {
+
+        if (typeof $ == 'function') {
+            var dlSignIsUsing = true;
+        } else {
+            var dlSignIsUsing = false;
+        }
+
+        function getScript(url, success) {
+            var script     = document.createElement('script');
+                script.src = url;
+
+            var head = document.getElementsByTagName('head')[0],
+                done = false;
+
+            // Attach handlers for all browsers
+            script.onload = script.onreadystatechange = function() {
+                if (! done && (!this.readyState || this.readyState == 'loaded' || this.readyState == 'complete')) {
+                    done = true;
 
-                });
+                    // callback function provided as param
+                    success();
+
+                    script.onload = script.onreadystatechange = null;
+                    head.removeChild(script);
+                };
+            };
+
+            head.appendChild(script);
+        };
+
+        getScript('https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js', function() {
```

Comment:
ok, I'll try to rewrite to remove those jQuery stuff,
I think it'd be better to not rely on 3rd-party lib. 👍